### ISSUE TITLE
Partner Themes: Primary colors in the subscription flow is inconsistent

### ISSUE THREAD

**Issue by taipeicoder** [Automattic/wp-calypso#93598]
## Description

The color switches from pink to blue depending on the screen.

![Image](https://github.com/user-attachments/assets/96be0aec-ef56-43e5-9c3b-dcb01abc8e09)

![Image](https://github.com/user-attachments/assets/5082de79-945e-4d63-8cbc-fa8e33f7bef2)

![Image](https://github.com/user-attachments/assets/abda5c59-1ce3-4b51-9643-83437877543f)

![Image](https://github.com/user-attachments/assets/096f0a4d-32ca-44cd-b222-4f9e62508325)

**Comment by davemart-in** [comment]
Let's just remove the pink and replace it with blueberry blue.

**Comment by mmtr** [comment]
> Let's just remove the pink and replace it with blueberry blue.

I feel that would make things more inconsistent with the rest of screens:

<img width="1717" alt="Screenshot 2024-08-19 at 14 40 20" src="https://github.com/user-attachments/assets/717908ff-dbbe-4123-8fdf-ebb4f1774e78">

**Comment by davemart-in** [comment]
Yeah, I guess it depends on what admin color scheme you have selected, hey? I thought this pink color was creeping in outside of the "Classic Dark" and "Classic Bright" admin color schemes being selected:

![CleanShot 2024-08-19 at 08 49 39@2x](https://github.com/user-attachments/assets/fc440c96-d0eb-4217-8728-28a2265e811c)

This issue should apply the admin’s selected primary color to these screens.

**Comment by davemart-in** [comment]
Related issue: https://github.com/Automattic/dotcom-forge/issues/8741

**Comment by davemart-in** [comment]
When a site is not selected, we should be using the blueberry blue color #3858e9 as the accent color.
